src/test/java/com/google/javascript/clutz/ProgramSubject.java

```diff
@@ -85,20 +88,13 @@ public void generatesDeclarations(String expected) {
     }
   }
 
-  public void reportsDiagnosticsContaining(String expectedDiagnostics) {
-    String unexpectedRes = null;
-    try {
-      unexpectedRes = parse();
-      failureStrategy.failComparing("expected diagnostics, but got a successful result",
-          expectedDiagnostics, unexpectedRes);
-    } catch (DeclarationGeneratorException e) {
-      assertThat(e.getMessage()).contains(expectedDiagnostics);
-    }
+  StringSubject diagnosticStream() {
+    return assertThat(parse(true));
   }
 
-  private String parse() throws AssertionError {
+  private String parse(boolean forDiagnostics) throws AssertionError {
```

Comment:
Yeah, I thought so too, but figured it's a private API. I changed this to return a two element `String[]`; as long as this is purely private to `ProgramSubject` I think that ought to be enough.